src/Serialization/PlutusData.purs

```diff
@@ -0,0 +1,67 @@
+module Serialization.PlutusData
+  ( convertPlutusData
+  ) where
+
+import Prelude
+
+import Data.BigInt as BigInt
+import Data.Map (Map)
+import Data.Map as Data.Map
+import Data.Maybe (Maybe)
+import Data.Traversable (for, traverse)
+import Data.Tuple (Tuple, fst, snd)
+import Data.Tuple.Nested (type (/\), (/\))
+
+import FfiHelpers (ContainerHelper, MaybeFfiHelper, containerHelper, maybeFfiHelper)
+import Serialization.BigNum (bigNumFromBigInt)
+import Serialization.Types (BigInt, BigNum, ConstrPlutusData, PlutusData, PlutusList, PlutusMap)
+import Types.ByteArray (ByteArray)
+import Types.PlutusData as T
+
+convertPlutusData :: T.PlutusData -> Maybe PlutusData
```

Comment:
Why is it partial? Shouldn't the types be isomorphic?

Reply:
It all boils down to partiality of `BigInt.from_str`.